### PR TITLE
new: usr: #36: Add functionality for DDL statements to add, drop and update views.

### PR DESCRIPTION
**Pull request by adeshr**

### CODE REVIEW

File: fatjdbc/src/test/java/com/qubole/quark/fatjdbc/test/DDLViewTest.java [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        + "values(\"warehouse_part\", \"Warehouse Partition\", 0, "
+        + "\"select * from canonical.public.warehouse as wr where wr.w_warehouse_sq_ft > 100\", "
+        + "1, 2, \"PUBLIC\")";
+    Connection connection = DriverManager.getConnection("jdbc:quark:fat:", props);
+    connection.createStatement().executeUpdate(sql1);
+    connection.close();
+  }
+
+  @Test
+  public void testAlterView() throws SQLException, ClassNotFoundException {
```
Comment (amoghmargoor) [inline fatjdbc/src/test/java/com/qubole/quark/fatjdbc/test/DDLViewTest.java]: Need one more negative test on Alter. Write a test 'Alter view ... where name = <some valid name>'. This should throw an error as only alter on id is allowed.

**Comment by amoghmargoor** [comment]
Overall looks good to me apart.
Just add one more negative test as requested.
There is a mistake in corresponding PR raised for incubator-calcite which I have pointed out.. https://github.com/qubole/incubator-calcite/pull/18. Please correct it, after which we can release a new version of calcite which needs to be referenced here in pom.xml.

**Comment by adeshr** [comment]
@amoghmargoor : I tried refractoring executecreate/View/DataSource method in last commit. Not much change in code size as of now, but we can see the impact during addition of cubes-create parser, as only one statement will be added in switch case block. If it's too complex, we can revert it.

**Comment by amoghmargoor** [comment]
Yeah, like you said not much of change in code size that too at the expense of readability... one reason for that is you tried refactoring between DataSource and View which do not share the fields... Refactoring between CreateView and AlterView or between CreateDatasource and AlterDatSource is required as they share the functionality.  But even that is not required currently. Lets keep it this way for now.

File: fatjdbc/src/test/java/com/qubole/quark/fatjdbc/test/DDLViewTest.java (hunk 1 above)
Comment (adeshr) [inline fatjdbc/src/test/java/com/qubole/quark/fatjdbc/test/DDLViewTest.java]: done

**Comment by amoghmargoor** [comment]
@adeshr Update the calcite version to 1.5.0-qds-r12 and rebase this with proper commit message.

**Pull request merged**